data/json/items/gun/50.json

```diff
@@ -109,5 +109,52 @@
     "magazine_well": 1,
     "magazines": [ [ "50", [ "as50mag" ] ] ],
     "flags": [ "NEVER_JAMS" ]
+  },
+  {
+    "id": "tac50",
+    "copy-from": "rifle_base",
+    "type": "GUN",
+    "name": "Mcmillan TAC-50",
+    "description": "Long-range anti-materiel, and anti-personnel sniper rifle made by Mcmillan Firearms. this weapon has made many records for longest kill range.",
```

Comment:
```suggestion
    "description": "A long-range anti-materiel, and anti-personnel sniper rifle made by McMillan Firearms, serving the Canadian Army since 2000 as the C15, and the Navy Seals as the Mk 15 Mod 0.  This .50 caliber bolt-action rifle is capable of defeating light vehicles, radar installations and crew served weapons at extreme distances.  It notably holds the longest range confirmed sniper kill, as well as the 4th and 5th longest.",
```